test/functional/rpc_net.py

```diff
@@ -51,6 +51,7 @@ def setup_network(self):
         self.disable_mocktime()
         self.setup_nodes()
         connect_nodes(self.nodes[0], 1)
+        connect_nodes(self.nodes[1], 0)
```

Comment:
you should simply drop 16898, we already merged it (and did this change, some other PR must have reverted)